### PR TITLE
S05 us18 user page score

### PR DESCRIPTION
**Pull request by PaulBOURY**

### CODE REVIEW

**Review by JadeJolivet — APPROVED** [review]

File: client/src/components/ModifyInfoModale/ModifyInfoModale.jsx
Comment (FGAT3) [inline client/src/components/ModifyInfoModale/ModifyInfoModale.jsx:1]: Need to be refacto with PJ's form

File: client/src/components/ModifyInfoModale/ModifyInfoModale.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  input {
+    @include form-input;
+    margin: 0.5rem;
+  }
+
+  button {
+    @include form-button;
+    padding: 0.25rem 1rem;
```
Comment (FGAT3) [inline client/src/components/ModifyInfoModale/ModifyInfoModale.scss]: Need to uniformize all buttons

File: client/src/components/ScoreDisplay/ScoreDisplay.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+function ScoreDisplay() {
+  return (
+    <>
+    <p className="scoreDisplay">Space Invaders</p> 
+    <p className="scoreDisplay"> 2000 points</p>
```
Comment (FGAT3) [inline client/src/components/ScoreDisplay/ScoreDisplay.jsx]: 1 p or 2 p ? To be or not to be ?

File: client/src/pages/ProfilePage/ProfilePage.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  };
+  const openModalDelete = () => setShowModalDelete(true);
+  const handleClickDelete = () => {
+    setShowModalDelete(false);
+  };
+
+  const [showModal, setShowModal] = useState(false);
+  const [selectedGame, setSelectedGame] = useState(null);
+
+  const openModal = (gameName) => {
```
Comment (FGAT3) [inline client/src/pages/ProfilePage/ProfilePage.jsx:24]: create file for this function and import it

File: client/src/pages/ProfilePage/ProfilePage.jsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    const normalizedGameName = gameName.toLowerCase();
+    const foundGame = gamesData.find(
+      (games) => games.name.toLowerCase() === normalizedGameName
+    );
+    if (!foundGame) {
+      return;
+    }
+
+    setSelectedGame({
```
Comment (FGAT3) [inline client/src/pages/ProfilePage/ProfilePage.jsx:33]: this should be a parameter for refacto

**Review by FGAT3 — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

File: client/src/components/ScoreDisplay/ScoreDisplay.jsx
Comment (Yoann-Fortin) [inline client/src/components/ScoreDisplay/ScoreDisplay.jsx:1]: You can see this page to implement an accessible table : https://www.w3.org/WAI/tutorials/tables/

File: client/src/pages/ProfilePage/ProfilePage.jsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  const openModalModify = () => setShowModalModify(true);
+  const handleClickModify = () => {
+    setShowModalModify(false);
+  };
+  const openModalDelete = () => setShowModalDelete(true);
+  const handleClickDelete = () => {
+    setShowModalDelete(false);
+  };
+
+  const [showModal, setShowModal] = useState(false);
```
Comment (Yoann-Fortin) [inline client/src/pages/ProfilePage/ProfilePage.jsx:17]: group your states please

File: client/src/pages/ProfilePage/ProfilePage.jsx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+import GameListModal from "../../components/GameListModal/GameListModal";
+
+import Avatar from "../../assets/images/avatar/Avatar-basic.svg";
+
+function ProfilePage() {
+  const gamesData = useLoaderData();
+  const [showModalModify, setShowModalModify] = useState(false);
+  const [showModalDelete, setShowModalDelete] = useState(false);
+
+  const openModalModify = () => setShowModalModify(true);
```
Comment (Yoann-Fortin) [inline client/src/pages/ProfilePage/ProfilePage.jsx]: you duplicate same logic, a function more generic with an inversion of state would be better

File: client/src/pages/ProfilePage/ProfilePage.jsx [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+  };
+
+  const closeModal = () => {
+    setShowModal(false);
+    setSelectedGame(null);
+  };
+
+  return (
+    <>
+      <h1>Profile</h1>
```
Comment (Yoann-Fortin) [inline client/src/pages/ProfilePage/ProfilePage.jsx]: extract your semantic in another component please

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Pull request merged**